project.clj

```diff
@@ -1,6 +1,6 @@
 (defproject witan.models.demography "0.1.0-SNAPSHOT"
-  :description "FIXME: write description"
-  :url "http://example.com/FIXME"
+  :description "witan.models.demography is a Clojure library to run demographic models."
```

Comment:
common mistake to say demography models rather than demographic models, so nice one @acron0